caluma/form/serializers.py

```diff
@@ -324,3 +333,82 @@ class SaveDocumentFloatAnswerSerializer(SaveAnswerSerializer):
 
     class Meta(SaveAnswerSerializer.Meta):
         pass
+
+
+class SaveDocumentTableAnswerSerializer(SaveAnswerSerializer):
+    value = serializers.GlobalIDPrimaryKeyRelatedField(
+        source="documents",
+        queryset=models.Document.objects,
+        many=True,
+        required=True,
+        help_text="List of document ids whereas one document is one row in the table",
```

Comment:
```suggestion
        help_text="List of document IDs, each representing one row in the table",
```

Reply:
or probably even better: "List of document IDs representing the rows in the table"

Reply:
Adjusted